crates/lang/codegen/src/generator/cross_calling.rs

```diff
@@ -140,7 +140,7 @@ impl CrossCalling<'_> {
         }
     }
 
-    /// Builds up the [`ink_env::call::ArgumentList`] type structure for the given types.
+    /// Builds up the `ink_env::call::utils::ArgumentList` type structure for the given types.
```

Comment:
`ink_env` is no dependency of this crate, hence intra-crate links to `ink_env` do not work.

Making it a dependency for this purpose seems kinda heavy and would also not work (I tried it in [this CI run](https://gitlab.parity.io/parity/ink/-/pipelines/116245)).